app/views/admin/shared/_sidenav_billing.html.haml

```diff
@@ -18,7 +18,7 @@
   %li.nav-header= Journal.model_name.human.pluralize
   = tab t_create_model(Journal), new_facility_journal_path, (sidenav_tab == 'new_journal')
   = tab t_manage_models(Journal), facility_journals_path, (sidenav_tab == 'journals')
-  - if current_facility.single_facility? && AccountManager.using_statements?
+  - if current_facility.single_facility? && Account.config.using_any_statements?
```

Comment:
`using_any_statements?` seems like somewhat of a clunky name. What about `statements_enabled?` or something like that?